### ISSUE TITLE
Hover over sliders to show modulation source

### ISSUE THREAD

**Issue by VincyZed** [surge-synthesizer/surge#2181]
So I just found this in an old Surge 1.2.1:

![Mod](https://user-images.githubusercontent.com/50145178/85238158-ffea5f80-b3f9-11ea-922d-a383752bef00.gif)

Something equivalent to that would be a great addition that would help tremendously to understand modulation in a patch!

**Comment by baconpaul** [comment]
This is such a great idea. Its too much plumbing to make 1.7.0 at this late stage but I definitely want to add it very soon.

**Comment by mkruselj** [comment]
Hmmm what happens if a target parameter has multiple assigned modulators?

**Comment by VincyZed** [comment]
It highlights all the assigned modulator to the parameter in question

**Comment by VincyZed** [comment]
Since highlight is already part of the routing bar when hovering over tabs, I presume (just a guess here) it could be pretty easy to implement that. And it would fit the theme perfectly and look very nice.

**Comment by baconpaul** [comment]
yeah it would not be hard. the only trick is the slider doesn't know the modulation sources so it has to message back to the synth to activate the hover (which is enough plumbing that I'm waiting until after we ship 170)

**Comment by mkruselj** [comment]
Do we want this as a togglable feature? I reckon with this enabled and gobs of modulation around, it could be quite epilepsy-inducing...
